aws-http-nio-client-netty/pom.xml

```diff
@@ -50,6 +50,8 @@
         <dependency>
             <groupId>io.netty</groupId>
             <artifactId>netty-transport-native-epoll</artifactId>
+            <version>${netty.version}</version>
+            <classifier>linux-x86_64</classifier>
```

Comment:
The SDK will still work out of the box, the simple, still a relatively performant variant. If customers want to tune it, they can, they just include an additional dependency based on their deployment architecture.